test/unit/app/controllers/network/network-contoller-test.js

```diff
@@ -1,9 +1,9 @@
 const assert = require('assert')
```

Comment:
This file has a typo in the name (`contoller` => `controller`)

Reply:
ahh thanks ill fix that